R/derive_extreme_records.R

```diff
@@ -95,6 +95,13 @@
 #'
 #'   *Permitted Values*: a condition
 #'
+#' @param keep_vars_source Variables to be kept in the new records
+#'
+#'   A named list returned by `exprs()` defining the variables to be kept for the
+#'   new records.
```

Comment:
tidyselect expressions should be accepted.

Reply:
Updated the documentation.